src/components/features/our-partner/index.jsx

```diff
@@ -1,8 +1,10 @@
 import React from "react";
 
-import Button from "../Button";
+import Button from "../../ui/button";
+import Section from "../../ui/section";
+import { attachClasses } from "../../../utils/index";
```

Comment:
1. index should not be present in the import statement.
2. Move this import after the classes import